src/mozanalysis/experiment.py

```diff
@@ -1227,23 +1219,26 @@ def _validate_last_date_data_required(self, attribute, value):
 class AnalysisWindow:
     """Represents the range of days in which to measure a metric.
 
-    The range is measured in "days after enrollment", and is inclusive.
+    The range is measured in "days relative enrollment", and is inclusive.
 
-    For example, ``AnalysisWindow(0, 6)`` is the first week after enrollment.
+    For example, ``AnalysisWindow(0, 6)`` is the first week after enrollment
+    and `AnalysisWindow(-8,-1)` is the week before enrollument
 
     Args:
-        start (int): First day of the analysis window, in days since
-            enrollment.
+        start (int): First day of the analysis window, in days relative
+            enrollment. 0 indicates the date of enrollment. Positive numbers
+            are after enrollment, negative are before.
         end (int): Final day of the analysis window, in days since
-            enrollment.
+            enrollment. 0 indicates the date of enrollment. Positive numbers
+            are after enrollment, negative are before.
     """
 
     start = attr.ib(type=int)
     end = attr.ib(type=int)
 
     @start.validator
     def _validate_start(self, attribute, value):
-        assert value >= 0
+        assert (value >= 0) or (value < 0 and self.end < 0)
 
     @end.validator
     def _validate_end(self, attribute, value):
```

Comment:
Should this also be the same sign as `start`?

Reply:
Good catch, I added a check for that.